File: EducationApp.DataAccessLayer/Models/Filters/User/UserFilterModel.cs
using EducationApp.DataAccessLayer.Helpers.Base;
using static EducationApp.DataAccessLayer.Entities.Enums.Enums;

namespace EducationApp.DataAccessLayer.Helpers
{
    public class UserFilterModel : BaseFilterModel
    {
        public UserSortType UsersSortType { get; set; }
        public UserFilterType UsersFilterType { get; set; }
    }
}

File: ClientApp/src/app/shared/models/user/UserFilterModel.ts
import { UserSortType } from "../../enums/UserSortType";
import { UsersFilterType } from "../../enums/UsersFilterType";
import { BaseFilterModel } from '../Base/BaseFilterModel';
import { SortType } from '../../enums/SortType';

export class UserFilterModel extends BaseFilterModel
{
    userSortType:UserSortType;
    userFilterStatus:UsersFilterType;
    sortType:SortType;
}